docs/src/main/asciidoc/scripting.adoc

```diff
@@ -33,7 +33,6 @@ Normally we would link to a Git repository to clone but in this case there is no
 //DEPS io.quarkus.platform:quarkus-bom:{quarkus-version}@pom
 //DEPS io.quarkus:quarkus-resteasy-reactive
 //JAVAC_OPTIONS -parameters
-//JAVA_OPTIONS -Djava.util.logging.manager=org.jboss.logmanager.LogManager
```

Comment:
Why did you remove that? I think we need it?

Reply:
`Logger` is not used in this code segment.

When used the guide specifically says that this line should be added see https://github.com/quarkusio/quarkus/pull/32028/files/8032ac025e82c5432520e330b22e124c851c37c3#diff-850f60be00e79955e6ac10aca753c6bc8f0d5c4f889860706983899c96ef2466R375.

Reply:
Even if you don't use a logger in your script, you have the standard Quarkus loggers logging things and they need this.

Reply:
> you have the standard Quarkus loggers logging things and they need this

According to the Quarkus docs:

https://github.com/quarkusio/quarkus/blob/21d598d70a9e62b619d69b3ddb95622af3755671/docs/src/main/asciidoc/logging.adoc?plain=1#L13

Reply:
the property is still recommended to set as otherwise your logging configuration is not going to take effect on the first lines.

Reply:
I added it back.